identity-service/src/routes/userEvents.js

```diff
@@ -0,0 +1,23 @@
+const { handleResponse, successResponse, errorResponseBadRequest } = require('../apiHelpers')
+const models = require('../models')
+
+module.exports = function (app) {
+  app.get('/userEvents', handleResponse(async (req, res) => {
+    const { walletAddress } = req.query
+    if (!walletAddress) {
+      return errorResponseBadRequest('Please provide a wallet address')
+    }
+    try {
+      const userEvents = await models.UserEvents.findOne({
+        where: {
+          walletAddress
+        }
+      })
+      return successResponse(userEvents)
```

Comment:
@raymondjacobson there's a chance someone could pass in an invalid wallet or the table doesn't contain the wallet record and this response would be null

Reply:
I was under the impression that findOne would throw if none were found, but added a check on !userEvents just in case.